src/templates/layouts/healthCareLocalFacility/index.tsx

```diff
@@ -22,13 +24,53 @@ export function HealthCareLocalFacility({
   path,
   administration,
   vamcEhrSystem,
+  officeHours,
+  address,
+  phoneNumber,
+  image,
+  facilityLocatorApiId,
+  geoLocation,
 }: FormattedHealthCareLocalFacility) {
   // Populate the side nav data for the side nav widget to fill in
   // Note: The side nav widget is in a separate app in the static-pages bundle
   useEffect(() => {
     window.sideNav = menu
   })
 
+  const structuredSchemaData = {
+    '@context': 'https://schema.org',
+    '@type': 'Place',
+    address: {
+      '@type': 'PostalAddress',
+      streetAddress: `${address.address_line1}${
+        address.address_line2 ? `, ${address.address_line2}` : ''
+      }`,
+      addressLocality: address.locality,
+      addressRegion: address.administrative_area,
+      postalCode: address.postal_code,
+    },
+    name: title,
+    telephone: phoneNumber,
+    openingHoursSpecification: officeHours.map((hours) => ({
+      '@type': 'OpeningHoursSpecification',
+      dayOfWeek: `https://schema.org/${dayOfWeek(hours.day)}`,
+      opens: numberToTimeString(hours.starthours),
+      closes: numberToTimeString(hours.endhours),
+    })),
+    hasMap: `https://maps.google.com?saddr=Current+Location&daddr=${encodeURIComponent(
+      `${address.address_line1}, ${address.locality}, ${address.postal_code}`
+    )}`,
+    // Shouldn't need all these optional chains, but because we're not
+    // validating data during runtime, just in case...
+    image: image?.links?.['2_1_large']?.href,
+    branchCode: facilityLocatorApiId,
+    geo: {
+      '@type': 'GeoCoordinates',
+      latitude: geoLocation.lat,
```

Comment:
So it looks like geoLocation can be null some times, that is why the tugboat is blowing up.
see: https://www.va.gov/lovell-federal-health-care-tricare/locations/naval-great-lakes-burkey-mall-pharmacy/
the geolocation is null here.

Reply:
![Screenshot 2025-04-11 at 2 30 55 PM](https://github.com/user-attachments/assets/97563283-971f-40ee-90eb-8322f39f40b4)

Reply:
<img width="916" alt="Screenshot 2025-04-11 at 2 31 37 PM" src="https://github.com/user-attachments/assets/87b41eba-65eb-4fb6-b143-717efe709c00" />